### DISCUSSION TITLE
Getting Console Error while Hover on Icon.

### DISCUSSION THREAD

**Discussion by JigarTanna1** [primefaces/community#1093] [PrimeNG]
we have used the p-table to display the records, and we have used to show the information by hover on row icon using p-overlayPanel popup. I am getting issue in console as below when I hover the last row of P-table. 

![issue](https://github.com/primefaces/community/assets/158574710/c808c25b-58c5-4e0a-a07c-42a9570d891e)

Need suggestion to resolve the issue.

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can identify the issue?

**Reply by JigarTanna1** [reply]
Hi, 
Thank you for the response please find the below link which contain the html code and typescript.
https://stackblitz.com/edit/angular-poheqv?file=src%2Findex.html,src%2Fmain.ts

**Reply by cetincakiroglu** [reply]
Hi,

Your example does not start and it doesn't include primeng as a dependency. Could you please update the link?

**Comment by cetincakiroglu** [comment]
Closing the discussion due to inactivity.
